INDEX

```diff
@@ -23,6 +23,15 @@
     "official": false,
     "primary_pkg": "drupal8"
   },
+  "duplicati": {
+    "MANIFEST": "duplicati.json",
+    "name": "Duplicati",
+    "description": "Duplicati 2.0 Store securely encrypted backups on cloud storage services!",
+    "official": false,
+    "icon": "https://raw.githubusercontent.com/ix-plugin-hub/iocage-plugin-index/master/icons/duplicati.png",
```

Comment:
Needs to be: https://icons.freenas.org/community-icons/duplicati.png

Reply:
Icon path updated.  I'll have to take a look at the permission denied.